requirements/run.txt

```diff
@@ -1 +1,3 @@
-ruamel.yaml
\ No newline at end of file
+python
```

Comment:
Is this really a valid requirement?

Reply:
I took inspiration for this from conda forge recipes.

Reply:
That is crazy if it is.

Reply:
We usually include python as a run requirement in conda recipes, right?

Reply:
yes, that is true. I wonder about whether this affects pip, though maybe we don't care :man_shrugging: